docs/csharp/quick-starts/arrays-and-collections.md

```diff
@@ -206,7 +206,7 @@ Save the file and type `dotnet run` to see the results.
 > To concentrate on just this section, you can comment out the code that calls `WorkingWithStrings();`. Just put two `/` characters in front of the call like this:  `// WorkingWithStrings();`. 
 
 ## Challenge
-See if you can put together some of the lessons from this and earlier lessons. Expand on what you've built so far with Fibonacci Numbers. Try and write the code to generate the first 20 numbers in the sequence.
+See if you can put together some of the lessons from this and earlier lessons. Expand on what you've built so far with Fibonacci Numbers. Try and write the code to generate the first 20 numbers in the sequence. (As a hint, the 20th Fibonacci number is 6765.)
```

Comment:
These are nits, since they're comments on the older content. But:
1. The repetition of "lessons" in the first sentence makes for a confusing sentence.
2. Try and write --> Try to write